tests/unit/components/fileLoader.test.js

```
@@ -67,11 +67,18 @@ describe('FileLoader', () => {
-    jest.clearAllMocks();
+    jest.restoreAllMocks();
     document.body.innerHTML = '';
     localStorage.clear();
   });
 
+  beforeEach(() => {
+    // Mock console methods to prevent test failures
+    jest.spyOn(console, 'error').mockImplementation(() => {});
+    jest.spyOn(console, 'warn').mockImplementation(() => {});
+    jest.spyOn(console, 'log').mockImplementation(() => {});
+  });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)